alpaca/data/models/news.py

```diff
@@ -17,6 +20,8 @@ class NewsImage(BaseModel):
     size: NewsImageSize
     url: str
 
+    model_config = ConfigDict(protected_namespaces=tuple())
```

Comment:
could you try to remove this? I have merged a PR for this. Therefore, doc generation process should work without this field.

Reply:
After removing this doc generation fails with
```
autodoc: failed to import class 'live.crypto.CryptoDataStream' from module 'alpaca.data'; the following exception was raised:
Traceback (most recent call last):
  File "/Users/gnvk/.pyenv/versions/alpaca-py/lib/python3.11/site-packages/sphinx/ext/autodoc/importer.py", line 143, in import_module
    return importlib.import_module(modname)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/gnvk/.pyenv/versions/3.11.5/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/gnvk/Dev/alpaca/clients/alpaca-py/alpaca/data/live/__init__.py", line 1, in <module>
    from alpaca.data.live.crypto import CryptoDataStream
  File "/Users/gnvk/Dev/alpaca/clients/alpaca-py/alpaca/data/live/crypto.py", line 5, in <module>
    from alpaca.data.live.websocket import DataStream
  File "/Users/gnvk/Dev/alpaca/clients/alpaca-py/alpaca/data/live/websocket.py", line 22, in <module>
    from alpaca.data.models.news import News
  File "/Users/gnvk/Dev/alpaca/clients/alpaca-py/alpaca/data/models/news.py", line 9, in <module>
    class NewsImage(BaseModel):
  File "/Users/gnvk/.pyenv/versions/alpaca-py/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 178, in __new__
    set_model_fields(cls, bases, config_wrapper, types_namespace)
  File "/Users/gnvk/.pyenv/versions/alpaca-py/lib/python3.11/site-packages/pydantic/_internal/_model_construction.py", line 452, in set_model_fields
    fields, class_vars = collect_model_fields(cls, bases, config_wrapper, types_namespace, typevars_map=typevars_map)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/gnvk/.pyenv/versions/alpaca-py/lib/python3.11/site-packages/pydantic/_internal/_fields.py", line 143, in collect_model_fields
    raise NameError(
NameError: Field "model_fields" conflicts with member {} of protected namespace "model_".

make: *** [html] Error 2
```

Reply:
Adding `import alpaca.data.models.news  # noqa # pylint: disable=unused-import` to docs/conf.py` fixes this.